2-mandatory/3-groceries.js

```diff
@@ -27,13 +27,28 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
```

Comment:
Again, try to use `const` over `let` if you are not resigning variables, and to ensure things are immutable :)